plugins/@grouparoo/sqlite/src/lib/table-import/getRows.ts

```diff
@@ -17,33 +14,29 @@ export const getRows: GetRowsMethod = async ({
   matchConditions,
   highWaterMarkKey,
   incremental,
-}: {
-  highWaterMarkCondition: MatchCondition;
-  [key: string]: any;
 }) => {
   // Begin with SELECT statement.
-  let query = `SELECT *, ${highWaterMarkAndSortColumnASC} AS ${highWaterMarkKey} FROM "${tableName}"`;
+  const queryBuilder = new SQLiteQueryBuilder(
+    `SELECT *, ${highWaterMarkAndSortColumnASC} AS ${highWaterMarkKey} FROM "${tableName}"`
```

Comment:
Do these column names need to be quoted?

Reply:
Good catch. I did another pass to ensure that column names are always quoted